mets_retriever/mets_retriever.py

```diff
@@ -74,6 +74,22 @@ def _verify_download(self, expected_path):
         if not os.path.isfile(expected_path):
             raise OSError("METS file not found at expected path after downloading")
 
+    def _get_replica_locations(self, replicas):
+        """Return list of AIP replica locations."""
+        replica_locations = []
+        for replica in replicas:
+            uuid = replica.replace("/api/v2/file/", "").rstrip("/")
```

Comment:
Nitpick: maybe the file API path could be a constant (as it's used in the tests as well but could change in the future).